lib/archethic/db/embedded_impl/inputs.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Archethic.DB.EmbeddedImpl.Inputs do
+  @moduledoc """
+  Inputs are stored by destination address. 1 file per address
+  There will be many files
+  """
+
+  alias Archethic.DB.EmbeddedImpl
+  alias Archethic.TransactionChain.VersionedTransactionInput
+  alias Archethic.Utils
+
+  @type ledger :: :token | :UCO
+
+  @spec append_inputs(
```

Comment:
A good thing would be to leverage a behavior in the `db.ex` file, so external contexts particularly during testing could leverage implementations as mock to simplify testing without relying on the file system. Hence, the `embedded_impl.ex` could be the facing API to the inputs offloading.